engines/rbd.c

```diff
@@ -529,6 +532,11 @@ static int fio_rbd_queue(struct thread_data *td, struct io_u *io_u)
 static int fio_rbd_init(struct thread_data *td)
 {
 	int r;
+	struct rbd_data *rbd;
+	rbd = (struct rbd_data*)(td->io_ops_data);
```

Comment:
You don't have to cast it, it's a void pointer. Just do:

struct rbd_data *rbd = td->io_ops->data;

like everywhere else in the code.